run_unittests.py

```diff
@@ -3404,6 +3404,12 @@ def test_templates(self):
                 langs.append('rust')
             except EnvironmentException:
                 pass
+        if is_osx():
```

Comment:
Everything looks good here except this, swift works on oses other than macos.

Reply:
So. Dose it work on Linux?

Reply:
Ok did some digging and turns out Swift can run on Linux.

Reply:
It’s not looking good doc. What do I do from here @dcbaker ?